### PR TITLE
overhaul some docs surrounding rusefi_config.txt

### PR DESCRIPTION
**Pull request by chuckwagoncomputing**
Review would be appreciated, this reflects my workflow, but it might not reflect everyone's.

### CODE REVIEW

File: firmware/rusefi.cpp [hunk 1]
```diff
@@ -84,17 +84,28 @@
  * 2.777ms, so we schedule spark firing at '2.777ms after the 690 position sensor event', thus combining trigger events
  * with time-based offset.
  *
+ * @section tunerstudio Getting Data To and From Tunerstudio
+ *
+ * Contains the enum with values to be output to Tunerstudio.
+ * console/binary/tunerstudio_outputs.h
+ * Tunerstudio output channels are handled completely manually, as opposed to settings where offsets are managed automatically by the gen_config scripts.
```
Comment (mck1117) [inline firmware/rusefi.cpp:91]: this is no longer true, have a look at `output_channels.txt`!

Comment (chuckwagoncomputing) [reply firmware/rusefi.cpp:91]: And offsets aren't handled manually anymore?

Comment (mck1117) [reply firmware/rusefi.cpp]: make a change in `output_channels.txt`, run `gen_output_channels.sh` and see the magic!

**Comment by chuckwagoncomputing** [comment]
@rusefillc I guess this is good to go if it looks good to you.

**Pull request merged**